WordPress/Tests/WP/DiscouragedConstantsUnitTest.inc

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * Make sure that global constants are correctly identified.
+ *
+ * The below are all ok.
+ */
+// Ok, not a constant.
+namespace STYLESHEETPATH {}
+namespace MY\OTHER\STYLESHEETPATH\NAMESPACE {}
+use STYLESHEETPATH;
+use Something, STYLESHEETPATH, SomethingElse;
+class STYLESHEETPATH {
+	const STYLESHEETPATH = 'something';
+	private function STYLESHEETPATH() {}
+}
+class ABC extends STYLESHEETPATH {}
+class DEF implements STYLESHEETPATH {}
+interface STYLESHEETPATH {}
+trait STYLESHEETPATH {}
+const STYLESHEETPATH = 'something';
```

Comment:
Good question. I added a lot of code to prevent false positives and only addressed the `define()` bit after that. Clearly forgot to also change things for `const`.

I'm going to change this.